typings/rendition.d.ts

```diff
@@ -450,6 +450,23 @@ declare module 'rendition' {
 
 	class Text extends RenderableElementWithProps<TextProps, any> {}
 
+	namespace Text {
```

Comment:
Why do we need this? If its required for some reason can you add a comment to the typing explaining its purpose please.

Reply:
Will do.
Without this the typings do not allow you to do `<Text.span>`. This is almost exactly what we did for `<Heading.h1,2,3,4...>` (even though the typings do not allow the plain `<Heading>` there and I'm not sure whether that's on purpose).